AppKit/CPViewController.j

```diff
@@ -176,6 +176,58 @@ var CPViewControllerCachedCibs;
         _view = [CPView new];
 }
 
+- (void)loadViewAsynchronously:(Function/*(view, error)*/)aHandler
```

Comment:
`loadViewWithCompletionHandler:(Function)aHandler` seems to be a winner because it is the most expressive, simple and consistent with cocoa.
If there is no objection I will rename the method and merge this PR.